lib/iris/util.py

```diff
@@ -41,6 +41,54 @@
 import iris.exceptions
 
 
+def _cube_cubes_arg(cubes):
+    """
+    Function for ensuring that we return a Cubelist, irrespective of whether
+    a Cube or a CubeList has been provided.
+
+    Args:
+
+    cubes (:class:`iris.cube.Cube` or :class:`iris.cube.CubeList`)
+
+    Returns:
+
+    :class:`iris.cube.CubeList`
+
+    """
+    # Function for ensuring that we always deal with a cubelists
+    if isinstance(cubes, iris.cube.Cube):
+        cubes = iris.cube.CubeList([cubes])
+    return cubes
+
+
+def invert_coordinate(cubes, coordinate):
+    """
+    In-place operation to reverse the direction of a coordinate in a cube or
```

Comment:
If this is going to be limited to dimension coordinates then it should say so. Have you considered allowing it to operate on 1-dimensional auxiliary coordinates as well?